charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -28,6 +28,8 @@ spec:
               value: "{{- printf "%s-%s" .Release.Name "s3-metadata" | trunc 63 | trimSuffix "-" -}}"
             - name: REDIS_HOST
               value: {{ template "redis.fullname" . }}
+            - name: ENDPOINT
```

Comment:
Unclear why this should be set when Orbit is enabled.

Reply:
I updated the endpoint to always have "Release.Name"-cloudserver-front. There is a PR to update the docker entry point script to accept csv style strings for this var.

Reply:
https://github.com/scality/S3/pull/1170

Reply:
Cool, thx! :+1: